les/api.go

```diff
@@ -352,3 +419,60 @@ func (api *PrivateLightAPI) GetCheckpointContractAddress() (string, error) {
 	}
 	return api.backend.oracle.config.Address.Hex(), nil
 }
+
+//API should be for light clients of les protocol
+type LightClientAPI struct {
+	le *LightEthereum
+}
+
+func NewLightClientAPI(le *LightEthereum) *LightClientAPI {
+	return &LightClientAPI{le}
+}
+
+// PullPeerGatewayFees updates cache by pulling gateway fee peer nodes
+func (api *LightClientAPI) GatewayFeeCache() map[string]*gatewayFeeEtherbase {
+	return api.le.handler.gatewayFeeCache.gatewayFeeMap
+}
+
+// PullPeerGatewayFees updates cache by pulling gateway fee peer nodes
+func (api *LightClientAPI) PullPeerGatewayFees() error {
```

Comment:
The complimentary word to `Broadcast` should be `Query` or `Request`, although this isnt a big deal and if you feel strongly, leave it.